easybuild/easyblocks/l/lammps.py

```diff
@@ -34,14 +34,15 @@
 import os
 import re
 import tempfile
+import copy
 from easybuild.tools import LooseVersion
 
 import easybuild.tools.environment as env
 import easybuild.tools.toolchain as toolchain
 from easybuild.framework.easyconfig import CUSTOM, MANDATORY
 from easybuild.tools.build_log import EasyBuildError, print_warning, print_msg
 from easybuild.tools.config import build_option
-from easybuild.tools.filetools import copy_dir
+from easybuild.tools.filetools import copy_dir, mkdir
 from easybuild.tools.modules import get_software_root, get_software_version
 from easybuild.tools.run import run_cmd
```

Comment:
run_shell_cmd? (or perhaps i misunderstand where we are at with the conversion to the new run commands, given that this wasn't used before either?